guide/onboarding/protecting-a-wallet.md

```diff
@@ -0,0 +1,223 @@
+
+### Touch ID/Face ID
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/onboarding/protecting-a-wallet/face-id-info.png"
+   retina = "/assets/images/guide/onboarding/protecting-a-wallet/face-id-info@2x.png"
+   width = 250
+   height = 250
+   alt-text = "Face ID screen"
+   caption = "Face ID UI from [Wallet UI Kit](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=1228%3A27860)"
+   layout = "float-left-desktop -background -shadow"
+%}
+
+One convenient way for users to verify their identity to unlock your wallet app is with biometrics. This could take the form of either touch or face ID.
+
+Users of Android and iOS devices are super familiar with these motions. More importantly, using biometrics does not require the user to remember anything, furthermore decreasing their chances of getting locked out of their wallet.
+
+Unfortunately, biometrics are not supported on all mobile devices. Hardcore security users may be wary about giving up their biometric information.
+
+</div>
+
+{% include /tip/open.html label="Tip: Be Transparent" icon="info" color="blue" %}
+
+Bitcoin wallet users may feel hesitant about the privacy implications of your product storing touch/face ID information. Be sure your app thoroughly relays how this information is stored.
+
+{% include /tip/close.html %}
+
+### PIN
+
+A PIN is one of the most common and traditional methods for securing a mobile application. This setup requires that users create a 4-6 digit PIN that must be entered when opening their wallet application, or when viewing sensitive information such as their recovery phrase.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/protecting-a-wallet/choose-pin.png"
+   retina = "/assets/images/guide/onboarding/protecting-a-wallet/choose-pin@2x.png"
+   alt-text = "Choosing a pin screen"
+   width = 250
+   height = 541
+   class = "shadow"
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/protecting-a-wallet/enter-pin.png"
+   retina = "/assets/images/guide/onboarding/protecting-a-wallet/enter-pin@2x.png"
+   alt-text = "Enter pin screen"
+   width = 250
+   height = 541
+   class = "shadow"
+%}
+
+</div>
+
+The PIN-based approaches can be supported by all devices, and therefore is more accessible. A PIN can also be used to encrypt data within a wallet, which makes any locally stored information on the device inaccessible. Additionally, some users prefer this type of security measure because they use the same or similiar PINs on multiple applications/devices, making it easy to remember.
```

Comment:
```suggestion
All devices can support the PIN-based approach, which makes it more accessible. A PIN can also be used to encrypt data within a wallet, making any locally stored information on the device inaccessible. Additionally, some users prefer this type of security measure because they use the same or similar PINs on multiple applications/devices, making it easy to remember.
```